File: python/2460.py
class Solution:
    def applyOperations(self, nums: List[int]) -> List[int]:
        for i in range(len(nums)-1):
            if nums[i] == nums[i+1]:
                nums[i] = nums[i]*2
                nums[i+1] = 0
        zer = nums.count(0)
        for i in range(zer):
            nums.pop(nums.index(0))
            nums.append(0)
        return nums

File: rust/2460.rs
impl Solution {
    pub fn apply_operations(nums: Vec<i32>) -> Vec<i32> {
        let mut nu = nums.clone();
        for i in 0..nu.len()-1 {
            if nu[i] == nu[i+1] {
                nu[i] = nu[i]*2;
                nu[i+1] = 0;
            }
        }
        let z = nu.iter().filter(|&v| *v == 0 ).count();
        for i in 0..z {
            nu.remove(nu.iter().position(|&x| x==0).expect("number"));
            nu.push(0);
        }
        nu
    }
}
